mlflow/utils/validation.py

```diff
@@ -80,7 +82,12 @@ def path_not_unique(name):
 
 def _validate_metric_name(name):
     """Check that `name` is a valid metric name and raise an exception if it isn't."""
-    if name is None or not _VALID_PARAM_AND_METRIC_NAMES.match(name):
+    if name is None:
+        raise MlflowException(
+            f"Invalid metric name: '{name}'. {_MISSING_KEY_NAME_MESSAGE}",
```

Comment:
This might look like `mlflow.log_metrics({"None": 0})` is disallowed.

Reply:
great observation. changed!